SU-SWS/saa_victoria_subtheme
# Adapt 714  icon card color tags

# READY

# Summary
- Fix the automatic color-assignment for tags on icon cards. 

# Review By (Date)
- 9/18 asap

# Urgency
- High - visible bug, launch is 9/21

# Steps to Test

1. Pull down the branch or look at this page that has the branch: https://saasierracamp-test.sites.stanford.edu/icon-card-test-page
2. Clear the site cache
3. Create a page with rows of icon cards: 1 card in a row, 1 card in a row, 1 card in a row
then 2 cards in a row, 2 cards in a row, 2 cards in a row.
Then 3 cards in a row, 3 cards in a row, 3 cards in a row.
Check that colors are rotating automatically.

Color tag order specs from Kerri:
here are the requests for color order (not sure if you need 3 in sequence):
**3 cards in a row**
yellow, green, blue
green, blue, yellow
blue, yellow, green
**2 cards in a row**
yellow, green
blue, yellow
green, blue
**1 card in a row**
yellow
green
blue


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
